app/post.go

```diff
@@ -1549,6 +1549,14 @@ func (a *App) GetRecentSearchesForUser(userID string) ([]*model.SearchParams, *m
 	return searchParams, nil
 }
 
+func (a *App) DeleteRecentSearchForUser(userID string, searchQuery *model.SearchParams) *model.AppError {
+	err := a.Srv().Store.Post().DeleteRecentSearchForUser(userID, searchQuery)
```

Comment:
Btw, when updating this to the latest master, this needs to be updated to
```suggestion
	err := a.Srv().Store().Post().DeleteRecentSearchForUser(userID, searchQuery)
```
because we changed how the stores are accessed